docs/src/main/tut/setup-pureconfig.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: docs
+title: Add PureConfig to your project
+---
+## {{page.title}}
+
+In the sbt configuration file use Scala `2.10`, `2.11` or `2.12`:
+
+```scala
+scalaVersion := "2.12.3" // or "2.11.11", "2.10.6"
+```
+
+Add PureConfig to your library dependencies. For Scala `2.11` and `2.12`:
+
+```scala
+libraryDependencies ++= Seq(
+  "com.github.pureconfig" %% "pureconfig" % "0.8.0"
+)
+```
+
+For Scala `2.10` you need also the Macro Paradise plugin:
+
+```scala
+libraryDependencies ++= Seq(
+  "com.github.pureconfig" %% "pureconfig" % "0.8.0",
+  compilerPlugin("org.scalamacros" % "paradise" % "2.0.1" cross CrossVersion.patch)
+)
+```
+
+For a full example of `build.sbt` you can have a look at this [build.sbt](https://github.com/melrief/pureconfig/blob/master/example/build.sbt)
```

Comment:
Nitpick: we ought to update this URL to use the new group name, https://github.com/pureconfig/pureconfig/blob/master/example/build.sbt

Reply:
It's time to get rid of all the references to melrief/pureconfig for real. I pushed c96d014 that should do that. The only references in the repository after that commit are the gitter ones.